src/index.js

```diff
@@ -4,12 +4,12 @@ import App from './App';
 import reportWebVitals from './reportWebVitals';
 import './styles/global.scss';
 
-ReactDOM.render(
-  <App />,
-  document.getElementById('root')
+
+ReactDOM.render(<App />, document.getElementById('root')
 );
```

Comment:
dlaczego to zamknięcie funkcji w tej lini?

Reply:
Szczerze mówiąc nie jestem chyba pewny o co chodzi